<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>558</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>558</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>558</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>969</width>
    <height>507</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>重力售卖机生产调试终端</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="display_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="display_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="display_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="display_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>校验：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QComboBox" name="port_list">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="baudrate_list">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="parity_list">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="data_bits_list">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>第4层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>第3层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>第2层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>第1层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tare_button_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>去皮</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_zero_button_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0点校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_2000_button_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_5000_button_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_5000_button_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_2000_button_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tare_button_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>去皮</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_zero_button_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0点校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_5000_button_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_2000_button_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tare_button_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>去皮</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_zero_button_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0点校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_5000_button_1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_2000_button_1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2000g校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tare_button_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>去皮</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibration_zero_button_1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0点校准</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="all_on_top_check_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>窗口总在最前面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="query_configration_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查询配置信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="configration_display">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>未知</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>配置信息：</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>使用说明</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
